canonical-web-and-design/jaas-dashboard
# Update color of icons to #E5E5E5

## Done

Update color of icons to `#E5E5E5`

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check color of icons in side nav is correct

## Details

Fixes #764
